### DISCUSSION TITLE
When will Michael Caisse's message library be open-sourced?

### DISCUSSION THREAD

**Discussion by bryceschober** [intel/compile-time-init-build#106] [Q&A] [answered]
@lukevalenty This is more of a rhetorical question, to which the answer is probably only "yes", not a date... But asking the question here give me an easy place to subscribe for the final answer. :smile:

**Comment by lukevalenty** [comment]
> @lukevalenty This is more of a rhetorical question, to which the answer is probably only "yes", not a date... But asking the question here give me an easy place to subscribe for the final answer. :smile: 

😄 I am working with him to get it in this repo as soon as possible. We are planning on using this repo in our production code and have internal deadlines.

While @mjcaisse-intel was kind enough to present it and has been making some changes, the library was originally written by me along with everything else in *cib*. 😉

**Comment by lukevalenty** [comment, marked as answer]
Hi @bryceschober! The msg library code is up, but the API is not yet stable and I am in the process of adding documentation and additional 
tests for cib + msg.

https://github.com/intel/compile-time-init-build/tree/main/include/msg

**Reply by lukevalenty** [reply]
Thanks for the feedback! I will be making improvements in the next PR. 😄
